paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundActivity.kt

```diff
@@ -91,17 +89,15 @@ internal class PaymentSheetPlaygroundActivity : AppCompatActivity(), ExternalPay
         super.onCreate(savedInstanceState)
 
         setContent {
-            val paymentSheet = rememberPaymentSheet(
-                paymentResultCallback = viewModel::onPaymentSheetResult,
-                createIntentCallback = viewModel::createIntentCallback,
-                externalPaymentMethodConfirmHandler = this,
-            )
-            val flowController = rememberPaymentSheetFlowController(
-                paymentOptionCallback = viewModel::onPaymentOptionSelected,
-                paymentResultCallback = viewModel::onPaymentSheetResult,
-                createIntentCallback = viewModel::createIntentCallback,
-                externalPaymentMethodConfirmHandler = this,
-            )
+            val paymentSheet = PaymentSheet.Builder(viewModel::onPaymentSheetResult)
```

Comment:
We use the "old" constructors in many other places. Might be worth updating all of those in a follow up.

Reply:
[Ticket](https://jira.corp.stripe.com/browse/MOBILESDK-2229)